### ISSUE TITLE
[1.11.2-0.8.2488] Miner does nothing after a certain point

### ISSUE THREAD

**Issue by jzburda** [ldtteam/minecolonies#1183]
### Minecolonies version

version 0.8.2488

### Expected behavior

- Miner should be mining

### Actual behaviour

- Miner sits at his workplace with a Solid-Placeholderblock in his hand and does nothing

### Steps to reproduce the problem

1. Build miner
2. Let miner mine for a bit
3. ...
4. Profit

http://i.imgur.com/YTpzs7S.png

**Comment by Raycoms** [comment]
Did you give him cobblestone?

2017-06-14 23:24 GMT-03:00 jzburda <notifications@github.com>:

**Comment by jzburda** [comment]
He had some in his inventory, yes.

**Comment by Asherslab** [comment]
Okay, i've had this problem before and i'm not sure which of the following fixed it (and if it was actually any of the following):
0. Check their Saturation (hunger) and if they have: A shovel, A Pick, Some Cobble, Some Planks (i think they have to be oak), Some slabs (i think, not sure on that one), some Fences, and some Torches.
1. Fire the miner, and then re-hire them (don't do it too quickly)
2. Fire the miner, and hire a different miner
3. Wait until one night/day cycle has finished
4. make sure his ladder/digging area doesn't have any unusual blocks in it (not sure how to describe what you'd be looking for sorry) 
5. Repair their hut 

Sorry if none of these help, i'll take a look and see if i notice anything weird with the miner. 

Thanks for being a part of the Community!

p.s forgot to mention, make sure the Pickaxe / shovel are the right level for his hut

**Comment by jzburda** [comment]
Asherslab: Just a small question, will firing a worker reset their level?

**Comment by Raycoms** [comment]
not if you hire him at the same job.

2017-06-14 23:56 GMT-03:00 jzburda <notifications@github.com>:

**Comment by lunadragnstar** [comment]
I've noticed that the miner does stop if he has been mining at the same level too long. I leveled up the miner and he continued to work after going down a couple levels.

**Comment by Thomasdavid097** [comment]
make sure he has food, deliveryman doesn't bring workers food atm.

**Comment by ldog** [comment]
In reply to Kreezxil:
 
Yes, the ones with the masks, if they are supposed to be in the Savanah, they are where they belong. I went to turn in a quest and a group of them showed up, who started attacking me, who I killed, and I noticed my rep going down each kill. This was my 1st trip to the town since after you added that mod. I haven't seen them again, granted I've only been back to town a couple times since.
It was a fair sized group but I don't think it was excessive, I'm only wearing normal iron armor and just a basic manyllin (sp?) broadsword and I was able to take them out. I'm not sure if they all came at once or spawned more like zombies, other than losing rep with town I  don't have any complaints about them. Think they're pretty cool actually. 
On a side note, all my torocraft shopkeepers vanished (along with Baily) and have been replaced by normal testificates. I think this happened when you pulled emergent villagers out, I keep forgetting about it.

And yes, all wood stuff needs to be oak.
